docs/source/operations/howto/backend-deployment.rst

```diff
@@ -0,0 +1,134 @@
+*****************************
+How to deploy Substra Backend
+*****************************
+
+This guide shows you how to deploy the backend component of Substra.
+
+Prerequisites
+=============
+
+To deploy a Substra backend you will need a fully configured Kubernetes cluster.
+You will also need to install `Helm <https://helm.sh/>`_.
+
+Preparing your Helm values
+==========================
+
+The Substra backend deployment is packaged using Helm.
+You can find the complete description of values that can be used to configure the chart on `Artifact Hub <https://artifacthub.io/packages/helm/substra/substra-backend>`_.
+
+To configure your values:
+
+#. Add the Helm repository for Substra:
+
+   .. code-block:: bash
+
+      helm repo add substra https://substra.github.io/charts/
+
+#. Create a Helm values file named ``backend-values.yaml`` with the following content:
+
+   .. code-block:: yaml
+
+      organizationName: ORGANIZATION
```

Comment:
For deploying multiple backend in a channel, should we just repeat this how-to guide with different `organizationName` or are there other steps?

Reply:
It's mostly that but some steps are still undocumented at the moment like how to connect two backend together for model exchange.

Reply:
I think it would be worthwhile to add those steps (provided it remains a reasonable effort). In most use cases multiple backends are deployed, so this additional part would definitely be very useful.

Reply:
Yes, it's just that I prefer to go step by step in order to have reasonable amounts of docs for you to review ! :)

Reply:
Makes sense, thank you!